auto_dev/dao/dummy_data.py

```diff
@@ -40,28 +39,28 @@ def generate_uuid() -> str:
 
 
 def generate_boolean() -> bool:
-    """Generates a random boolean value."""
-    return random.choice(["True", "False"])  # noqa: S311
+    """Generates a deterministic boolean value."""
+    return True
 
 
 def generate_string() -> str:
-    """Generates a generic fake string."""
-    return fake.word()
+    """Generates a deterministic string."""
+    return "STRING_VALUE"
 
 
 def generate_integer() -> int:
```

Comment:
This is already present in a the contract stuff, so you can go from a type to a random value